src/scss/components/_track-key.scss

```diff
@@ -92,6 +92,18 @@
   &:focus {
     text-decoration: underline;
   }
+
+  &::before {
+    $size: 12px;
+    position: absolute;
+    top: 50%;
+    right: 10px;
```

Comment:
Probably best to use `$react-timelines-sidebar-key-indent-width` i.e. 20px. Need to check what happens when text runs into image too (i.e. do we need to add a matching `pad-right` to label to accomodate).